README.md

```diff
@@ -273,6 +273,17 @@ Contributions are what make the open source community such an amazing place to b
 12. Click "Done".
 13. You're good to go. Now you can easily add your Zoom integration in the Cal.com settings.
 
+## Setting up Stripe
```

Comment:
let's move this and all other pro feature instructions into `/ee/README.md`

Reply:
Done!